src/test/java/com/artipie/rpm/misc/FileInDirTest.java

```diff
@@ -46,4 +47,22 @@ void findsFile(@TempDir final Path tmp) throws IOException {
             new IsEqual<>(file)
         );
     }
+
+    @Test
+    void doesNotFindFile(@TempDir final Path tmp) throws IOException {
+        Files.write(tmp.resolve("a_file.txt"), "abc123".getBytes());
+        Assertions.assertThrows(
+            IllegalArgumentException.class,
+            () -> new FileInDir(tmp).find("_notfile.t")
+        );
+    }
+
+    @Test
+    void doesNotUseRegexToFind(@TempDir final Path tmp) throws IOException {
```

Comment:
@victornoel Why `FileInDir` shouldn't use regex to find a file?

Reply:
@paulodamaso I don't know, that's how it is currently implemented at least. If this should change, we can add a todo if you want? Not sure it is desired though…